root/doc/edit_type.tt

```diff
@@ -15,7 +15,7 @@
    <h1>[% class.l_edit_name %]</h1>
 
    <p>
-   <a href="[% c.uri_for_action('/edit/edit_types') %]">[% l('Edit types') %]</a> &gt;
+   <a href="[% c.uri_for_action('/edit/edit_types') %]">[% lp('Edit types', 'noun') %]</a> &gt;
```

Comment:
I was wondering if we should add the same to "Edit note".

Reply:
It probably makes sense, yes.